README.md

```diff
@@ -34,4 +34,25 @@ that's built into gxi, so please don't use `cargo` to install gxi, as that won't
 from a different rev can lead to very weird bugs.
 
 
-After these steps you should be able to run gxi simply by invoking `gxi`
\ No newline at end of file
+After these steps you should be able to run gxi simply by invoking `gxi`
+
+### Installation on Arch/Manjaro
+
+The package for Arch Linux is [gxi-git](https://aur.archlinux.org/packages/gxi-git/). Building and installing
```

Comment:
Hm, that sounds like gxi is in the main repos already. Maybe something like ```gxi can be [found in the AUR](link)``` would be better?

Reply:
Good point.